timeline-chart/src/layer/time-graph-chart.ts

```diff
@@ -214,7 +221,7 @@ export class TimeGraphChart extends TimeGraphChartLayer {
         this.stage.on('mouseup', mouseUpHandler);
```

Comment:
The mousemove handler added to the stage does not get removed and still gets called after closing the time graph. I would guess all the other stage handlers aren't removed either, however they aren't being triggered when their time graph is closed.

Reply:
None of the stage listeners get removed. Or is it that the stage itself needs to be destroyed (along with all its listeners)?

Reply:
I'll remove all the stage listeners at destroy, as well, plus destroy the pixi application when closing.